browser/ipfs/ipfs_tab_helper.cc

```diff
@@ -43,6 +50,30 @@ const char kDnsDomainPrefix[] = "_dnslink.";
 // The value of the header is the IPFS path of the returned payload.
 const char kIfpsPathHeader[] = "x-ipfs-path";
 
+#if !BUILDFLAG(IS_ANDROID)
+const char kIpfsFallbackOriginalUrlKey[] = "ipfs-fallback-original-url";
+const char kIpfsFallbackBlockRedirectForNavEntryIdKey[] =
+    "ipfs-fallback-block-redirect-for-nav-entry-id";
+struct IpfsFallbackOriginalUrlData : public base::SupportsUserData::Data {
```

Comment:
pls add new lines between vars and structs/classes.

Reply:
done